src/ide/completion/path.rs

```diff
@@ -108,11 +119,12 @@ pub fn path_suffix_completions<'db>(
             } else {
                 None
             };
+
             let importable_crate = importable_crate_id(db, *importable);
             let is_current_crate = importable_crate == current_crate;
             let is_core = *importable_crate.long(db) == CrateLongId::core(db);
 
-            let struct_initialization_text = match importable {
+            let struct_initialization_snippet_text = match importable {
```

Comment:
Perhaps we should calculate snippet text here instead of moving it around and checking if it's `None`